ui/Screen/Org.svelte

```diff
@@ -89,6 +89,13 @@
       },
     ];
   };
+
+  let pendingAnchor: boolean = false;
+  $: {
+    pendingAnchor = activeTab.anchoredProjects.some(
+      proj => proj.anchor.type === "pending"
```

Comment:
We should match against the type, not an arbitrary string here.

```suggestion
      proj => proj.anchor.type === project.PendingAnchor
```

And import the type from project at the top:
```
import type * as project from "ui/src/project"
```

Reply:
Yea, disregard this. It doesn't make sense.